submissions/krylenger/html-css-popup/style.css

```diff
@@ -0,0 +1,245 @@
+* {
+    font-family: monospace, Helvetica, sans-serif, Arial;
+    font-size: 17px;
+    font-weight: 500;
+    margin: 0px;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+a {
+    text-decoration: none;
+    outline: none;
+}
+
+img {
+    width: 75%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+span {
+    text-align: center;
+}
+
+label {
+    width: 100%;
+}
+
+label:hover {
+    cursor: pointer;
+}
+
+.navigation-container__sub-menu .checkbox {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  background-color: #eee;
+  opacity: 0;
+}
+
+.checkbox:hover {
+    cursor: pointer;
+}
+
+.navigation-list-icons {
+    background-size: 35px 35px;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+}
+
+.toggle-pop-up-label {
+    background-image: url(./icons/menu.png);
+}
+
+.navigation-notification {
+    background-image: url(./icons/notification.png);
+}
+
+.navigation-profile {
+    background-image: url(./icons/profile.jpg);
+}
+
+.toggle-pop-up:focus + label {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-container__list-of-items > li > a:focus {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-container { 
+    display: flex;
+    width: 100%;
+    height: 50px;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.navigation-container__body {
+    width: 320px;
+    height: 100%;
+}
+
+.pop-up-container {
+    display: none;
+    text-align: center;
+    height: 420px;
+    width: 270px;
+    border: solid 1px lightgrey;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px 0px rgba(0,0,0,1);
+}
+
+.pop-up-icons-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+    height: 94%;
```

Comment:
Maybe it's better to use some padding here instead of the 'magic number` of `94%`

Reply:
done ✅